server/src/main.go

```diff
@@ -1426,6 +1439,38 @@ func (c MainController) SupporterSpokeCSVHandler(w http.ResponseWriter, r *http.
 	writer.Flush()
 }
 
+func (c MainController) NewSupporterSpokeCSVHandler(w http.ResponseWriter, r *http.Request) {
+	chapter := getAuthedADBChapter(c.db, r)
+
+	startDate := r.URL.Query().Get("start_date")
+	endDate := r.URL.Query().Get("end_date")
+
+	activists, err := model.GetNewSupporterSpokeInfo(c.db, chapter, startDate, endDate)
```

Comment:
can you rename to GetNewActivistsSpokeInfo? looks like it's not building anyway :)